x/ecocredit/server/core/features/msg_add_credit_type.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Msg/AddCreditType
```

Comment:
i think we also need the test case for attempting to add credit type with duplicate abbreviation. the abbreviation is the primary key, name is the unique index